libraries/core_libs/network/src/tarcap/packets_handlers/status_packet_handler.cpp

```diff
@@ -73,17 +73,6 @@ void StatusPacketHandler::process(const PacketData& packet_data, const std::shar
       }
     }
 
-    // We need logic when some different node versions might still be compatible
-    if (node_major_version != TARAXA_MAJOR_VERSION || node_minor_version != TARAXA_MINOR_VERSION) {
-      // Log this only if we have 0 peers, so it is error of this node
-      LOG((peers_state_->getPeersCount()) ? log_nf_ : log_er_)
-          << "Incorrect node version: "
-          << getFormattedVersion({node_major_version, node_minor_version, node_patch_version}) << ", our node version"
-          << TARAXA_VERSION << ", host " << packet_data.from_node_id_.abridged() << " will be disconnected";
-      disconnect(packet_data.from_node_id_, dev::p2p::UserReason);
-      return;
-    }
```

Comment:
We don't check node version anymore?

Reply:
Yes. We will work with node that supports the same version of capability
For example node1 has capabilities [1,2] and node2 has [2,3]. So they can communicate with capability with version 2

Reply:
Could you please show me where the code do the check? Thanks

Reply:
Here we removing all capabilities that can't be used [Host.cpp:239](https://github.com/Taraxa-project/taraxa-node/blob/cc087eee3e366acafe8b2851d9ba8efb4bbe34da/libraries/aleth/libp2p/Host.cpp#L239). In this line [Host.cpp:275](https://github.com/Taraxa-project/taraxa-node/blob/cc087eee3e366acafe8b2851d9ba8efb4bbe34da/libraries/aleth/libp2p/Host.cpp#L275) we are disconnecting if there is no common capability

Reply:
Thank you